cmd/lakefs/cmd/kv.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/kv"
+	"github.com/treeverse/lakefs/pkg/logging"
+	"github.com/treeverse/lakefs/pkg/version"
+)
+
+const (
+	ScanCmdMinArgs = 1
+	ScanCmdMaxArgs = 2
+	GetCmdNumArgs  = 2
+)
+
+var kvCmd = &cobra.Command{
+	Use:    "kv",
+	Short:  "Inspect lakeFS' Key-Value Store",
+	Hidden: true,
+}
+
+var kvGetCmd = &cobra.Command{
+	Use:    "get <partition key> <key>",
+	Short:  "Return the value for the given key under the given partition",
+	Hidden: true,
+	Args:   cobra.ExactArgs(GetCmdNumArgs),
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := logging.Default()
+		cfg := loadConfig()
+
+		ctx := cmd.Context()
+		logger.WithField("version", version.Version).Info("lakeFS kv")
```

Comment:
Can be removed - we have a command that print it out if needed.

Reply:
Removed